packages/sdk/client/src/packlets/tests/invitations.test.ts

```diff
@@ -353,12 +353,10 @@ describe('Invitations', () => {
       afterTest(() => Promise.all([guest.destroy()]));
     });
 
+    // TODO(wittjosiah): `any` should be `LocalClientServices`.
     testSuite(
       () => ({ host: host.halo, guest: guest.halo }),
-      () => [
-        (host.services as LocalClientServices).host._serviceContext,
-        (guest.services as LocalClientServices).host._serviceContext,
-      ],
+      () => [(host.services as any).host._serviceContext, (guest.services as any).host._serviceContext],
```

Comment:
Not sure what's going on here, without this I was getting 
![image](https://github.com/dxos/dxos/assets/4529818/6f6081fe-54aa-425c-9190-0e60e31f4c64)
event though it definitely does exist https://github.com/dxos/dxos/blob/wittjosiah/05-25-refactor_factor_out_client_protocol/packages/sdk/client-services/src/packlets/services/service-host.ts#L72